client/src/pages/Login.jsx

```diff
@@ -1,100 +1,124 @@
 import React, { useState } from "react";
 // import { useHistory } from "react-router-dom";
-import { FormGroup, Stack } from "@mui/material";
+import LockIcon from "@mui/icons-material/Lock";
+import { Avatar, Box, Button, FormControl, Stack, TextField, Typography } from "@mui/material";
 //import { useState } from "react";
-import { useNavigation } from "react-router-dom";
 import { useMutation } from "@apollo/client";
+import { NavLink } from "react-router-dom";
+import { Copyright } from "../components/Copyright";
+import useAuthService from "../utils/authHook";
 import { LOGIN_USER } from "../utils/mutations";
-import Auth from "../utils/auth";
-export default function Login(){
-    const[userFormData, setUserFormData] = useState({email: '', password: ''});
-    const [validated] = useState(false);
-    const [showAlert, setShowAlert] = useState(false);
-    const handleInputChange = (event) => {
-        console.log("input change")
-        const {name, value} = event.target;
-        setUserFormData({
-            ...userFormData,
-            [name]: value,
-        });
+export default function Login() {
+  const [userFormData, setUserFormData] = useState({ email: "email@domain.com", password: "Password12345" });
```

Comment:
Can you explain these values please?

Reply:
looks like default information to, it is slow to be typing this in everytime to test changes in development.
this can be removed once that component is ready or complete.